services/services.go

```diff
@@ -904,6 +985,17 @@ func LatestBurnData() *types.BurnPageData {
 	return &types.BurnPageData{}
 }
 
+func LatestEthStoreStatistics() *types.EthStoreStatistics {
+	wanted := &types.EthStoreStatistics{}
+	cacheKey := fmt.Sprintf("%d:frontend:ethStoreStatistics", utils.Config.Chain.Config.DepositChainID)
+	if wanted, err := cache.TieredCache.GetWithLocalTimeout(cacheKey, time.Second*60, wanted); err == nil {
+		return wanted.(*types.EthStoreStatistics)
+	} else {
+		logger.Errorf("error retrieving burn data from cache: %v", err)
+	}
+	return wanted
```

Comment:
Shouldn't this return `&types.EthStoreStatistics{}`?

Reply:
Don't know if this makes any differences, but fixed to keep in line with similar functions